bnd.bnd

```diff
@@ -0,0 +1,28 @@
+Export-Package:\
+  org.apache.sling.launchpad.testservices.exported,\
+  org.apache.sling.launchpad.testservices.models,\
+  org.apache.sling.launchpad.testservices.events
+
+Private-Package:\
```

Comment:
No need for this non-standard header

Reply:
Thanks for the review.  This was a direct copy of how the old maven-bundle-plugin instructions were doing it.   I don't have a problem removing this instruction as it doesn't appear to have any impact on running the tests.

However, I would note that removing this instruction doesn't prevent this attribute from showing up in the generated manifest as bnd would just use the default configuration.

I've seen a few places using the "removeheaders" instruction to remove it from the output, is this what you were expecting?

-removeheaders:\
  Private-Package

Reply:
No need for removing, just go with the standard value for `Private-Package` as generated by bnd.

Reply:
fair enough.  This was already done with the last commit.